ui/apps/platform/src/Containers/NetworkGraph/components/LegendContent.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Flex, FlexItem, Title } from '@patternfly/react-core';
+import { PficonNetworkRangeIcon, BuilderImageIcon } from '@patternfly/react-icons';
+
+import DescriptionListItem from 'Components/DescriptionListItem';
+import DescriptionListCompact from 'Components/DescriptionListCompact';
+
+import { ReactComponent as BothPolicyRules } from 'images/network-graph/both-policy-rules.svg';
+import { ReactComponent as EgressOnly } from 'images/network-graph/egress-only.svg';
+import { ReactComponent as IngressOnly } from 'images/network-graph/ingress-only.svg';
+import { ReactComponent as NoPolicyRules } from 'images/network-graph/no-policy-rules.svg';
+
+function LegendContent() {
+    return (
+        <Flex direction={{ default: 'column' }}>
+            <FlexItem>
+                <Title headingLevel="h4" className="pf-u-pb-sm">
+                    Node types
+                </Title>
+                <DescriptionListCompact isHorizontal termWidth="20px" className="pf-u-pl-md">
+                    <DescriptionListItem term={<BuilderImageIcon />} desc="Deployment" />
+                    <DescriptionListItem
+                        term={<PficonNetworkRangeIcon />}
+                        desc="External CIDR block"
+                    />
+                </DescriptionListCompact>
+            </FlexItem>
+            <FlexItem>
+                <Title headingLevel="h4" className="pf-u-pb-sm">
```

Comment:
I would just double-check to see if the heading level is correct (not sure if it is from the code diff)

Reply:
This may be a use-case for a hidden heading in the pop-up that reads "Legend", and has a heading level one higher than these.

```
   <Title headingLevel="h3" className="pf-u-screen-reader">Legend</Title>
```